LDM-533.tex

```diff
@@ -231,7 +236,17 @@ \subsection{Suspension criteria and resumption requirements}
 
 \subsection{Naming convention}
 
-All tests are named according to the pattern \textsc{prod-xx-yy} where:
+With the introduction of the Jira Test Management, the following definitions have to be considered:
+
+\begin{description}
+  \item[LVV]{: Is the label for the ``LSST Verification and Validation'' project in Jira where all information regarding tests are managed.}
+  \item[LVV-XXX]{: Are Verification Elements, where XXX is the Verification Element identifier.  Each Verification Element is derived from a requirement and has at least one Test Case associated. There can be multiple Verification Elements associated with a requirement.}
+  \item[LVV-TYYY]{: Are Test Cases. Each Test Case is associated with a Verification Element, where YYY is the Test Case identifier. There can be multiple test cases associated with a Verification Element.}
+\end{description}
+
+The old naming convention described bellow is now obsolete, but existing test cases can be encountred in old documents, and will keep the original identifier at the beginnig of the name.
```

Comment:
"bellow" -> "below", "encountred" -> "encountered", "beginnig" -> "beginning"

Reply:
Typos corrected. Thank you